all/plugin-dialects/README.md

```diff
@@ -2,10 +2,178 @@
 
 FAW plugin for partitioning the corpus into dialects.
 
+
+## Motivation
+
+The dialects wizard is a tool for finding mutually disjoint features that cover
+the corpus, or a subset of it. Like the clustering plugin, it is intended for
+root cause analysis, i.e. finding the relationships between interesting
+features. Unlike the clustering plugin, which primarily deals only with pairs
+of features (making its name a misnomer), the dialect wizard facilitates dealing
+with groups of features.
+
+The dialect wizard can answer questions like:
+* "What kinds of features are produced by running parsers on this corpus? Are
+  most of my files similar to each other, or are there various categories
+  of files?"
+* "Are the files that produced this error all similar, or can we
+  partition them?"
+* "What factors might have caused this feature?"
```

Comment:
This might actually be a bit of a weakness of the dialect wizard at present -- I think it only finds features that might have caused something *if those features are mutually exclusive*. Otherwise, I think it would count as outliers, which the current plugin isn't great at finding. Could always add a "do not count files in multiple partitions as outliers" feature to handle this use case?

Reply:
And in meantime, just document this quirk, and that it could be fixed in a future version